internal/printer/event_test.go

```diff
@@ -176,6 +176,24 @@ func Test_ReplicaSetEvents(t *testing.T) {
 				FirstTimestamp: metav1.Time{Time: time.Unix(1548424410, 0)},
 				LastTimestamp:  metav1.Time{Time: time.Unix(1548424410, 0)},
 			},
+			{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "frontend",
+					CreationTimestamp: metav1.Time{
+						Time: now,
+					},
+					ResourceVersion: "3",
+				},
+				Count:  1,
+				Type:   corev1.EventTypeNormal,
+				Reason: "SuccessfulCreate",
+				Source: corev1.EventSource{
+					Component: "replicaset-controller",
+				},
+				Message:        "Created pod: frontend-b7fxg",
+				FirstTimestamp: metav1.Time{Time: time.Time{}},
```

Comment:
Since the struct field is optional, omit `FirstTimestamp`/`LastTimestamp` entirely. After the conversion to int64, it'll be some really large negative number. Passing this value into `relative.pipe.spec.ts` reproduces the problematic timestamp seen in the issue.

Reply:
Okay so I will omit it entirely.